packages/react/spec/useBulkAction.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { GadgetRecord } from "@gadgetinc/api-client-core";
+import { CombinedError } from "@urql/core";
+import { assert, IsExact } from "conditional-type-checks";
+import { useBulkAction } from "../src";
+import { bulkExampleApi } from "./apis";
+
+// these functions are typechecked but never run to avoid actually making API calls
+const TestUseBulkActionCanRunActionsWithVariables = () => {
```

Comment:
Thanks for writing all of these tests. It was on my backlog to tackle this month, alongside routes work 🙇

Reply:
No problemo!